.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["denoland.vscode-deno"]
```

Comment:
alot of VScode specification seems to be added in this commit, can you add .vscode in ignore aswell?